tests/test_flows.py

```diff
@@ -119,6 +133,11 @@ def successful_job_path(request, route):
         )
 
 
+class JobSubmissionDummyHandler(object):
+    def handle(self, result):
+        self.result = result
+
+
```

Comment:
we can nix this now right? seems unused at this point

Reply:
Fixed with [`51c1de9` (#89)](https://github.com/PrefectHQ/prefect-databricks/pull/89/commits/51c1de9791dd88574451c9fbbefc018cca1e8d29)